crates/sputnik/README.md

```diff
@@ -0,0 +1,23 @@
+# 🚀 Sputnik
+
+Sputnik is a crate to aid in the collection of anonymous usage data for CLIs built in Rust.
+
+## Overview
+
+Sputnik declares the `sputnik::Session` struct which is used to capture information about a single command execution. example:
+
+```sh
+body:
+  command:     {"command":{"Config":{"command":{"Profile":{"command":"List"}}}}}
```

Comment:
This structure will be hard to search / index within our backend. Does this only capture arguments and not flags? If so, why not a `vec<String>` or something more simple where the depth isn't variable?

Reply:
this will capture _anything_ that is part of our `structopt` fields that do not have `#[serde(skip_serializing)` on it, so it should capture _all_ information we want. we can also write custom seriazliers for different commands to override the defaults, though i do not have an example of how to do that which i can take as an action item